code/game/objects/items/weapons/storage/lockbox.dm

```diff
@@ -52,6 +52,13 @@
 			..()
 		return
 
+/obj/item/weapon/storage/lockbox/can_be_inserted(obj/item/W as obj, stop_messages = 0)
+	if(!(locked))

```

Comment:
the inner `()` braces of this make it less readable, `if(!locked)` and `if(!stop_messages)` would work fine